### ISSUE TITLE
Support for Lab test runner

### ISSUE THREAD

**Issue by ltegman** [skovhus/jest-codemods#84]
I would love a codemod to help transition from https://github.com/hapijs/lab. I'd be happy to look into doing it myself when I get the time, but I just wanted an issue out here for tracking if you don't mind.

**Comment by skovhus** [comment]
Thanks @ltegman. It is actually two test frameworks I've never worked with... 👍 

Do you have a large test set using these? It really makes writing the codemod more fun and motivating. : )

`code` should be able to leverage a lot of the work done in the `chai` transformers.

**Comment by ltegman** [comment]
Yeah, we've got multiple projects with anywhere from 100 - 1000+ tests in them, so I'll have a large test case to work with! That was exactly my motivation for wanting this transform, because I'd love to give jest a try, but no interest is doing that migration by hand! :sweat_smile: :

I'm only just now writing my first codemod for another internal library migration we're doing, but once I'm done with that I'd love to try my hand at this :smile:

**Comment by skovhus** [comment]
@ltegman how did it go with your first codemod? Let me know if you still are up for contributing here. Thanks : )
